src/Forms/EffectKaraoke.cs

```diff
@@ -330,17 +345,8 @@ private void MakeAnimation()
                 }
                 i++;
             }
-
-            if (numericUpDownDelay.Value > 0)
-            {
-                startMilliseconds = index * stepsLength;
-                startMilliseconds += _paragraph.StartTime.TotalMilliseconds;
-                endMilliseconds = _paragraph.EndTime.TotalMilliseconds;
-                start = new TimeCode(startMilliseconds);
-                end = new TimeCode(endMilliseconds);
-                _animation.Add(new Paragraph(start, end, startFontTag + _paragraph.Text + endFontTag));
-            }

```

Comment:
```
1101
01:04:43,380 --> 01:04:43,493
<font color="#ff0000">H</font>eh. And you knew everything.

1102
01:04:43,494 --> 01:04:43,607
<font color="#ff0000">He</font>h. And you knew everything.

1103
01:04:43,608 --> 01:04:43,722
<font color="#ff0000">Heh</font>. And you knew everything.

1104
01:04:43,723 --> 01:04:43,836
<font color="#ff0000">Heh.</font> And you knew everything.

1105
01:04:43,837 --> 01:04:43,950
<font color="#ff0000">Heh. </font>And you knew everything.

1106
01:04:43,951 --> 01:04:44,064
<font color="#ff0000">Heh. A</font>nd you knew everything.

1107
01:04:44,065 --> 01:04:44,178
<font color="#ff0000">Heh. An</font>d you knew everything.

1108
01:04:44,179 --> 01:04:44,292
<font color="#ff0000">Heh. And</font> you knew everything.

1109
01:04:44,293 --> 01:04:44,407
<font color="#ff0000">Heh. And </font>you knew everything.

1110
01:04:44,408 --> 01:04:44,521
<font color="#ff0000">Heh. And y</font>ou knew everything.

1111
01:04:44,522 --> 01:04:44,635
<font color="#ff0000">Heh. And yo</font>u knew everything.

1112
01:04:44,636 --> 01:04:44,749
<font color="#ff0000">Heh. And you</font> knew everything.

1113
01:04:44,750 --> 01:04:44,863
<font color="#ff0000">Heh. And you </font>knew everything.

1114
01:04:44,864 --> 01:04:44,977
<font color="#ff0000">Heh. And you k</font>new everything.

1115
01:04:44,978 --> 01:04:45,092
<font color="#ff0000">Heh. And you kn</font>ew everything.

1116
01:04:45,093 --> 01:04:45,206
<font color="#ff0000">Heh. And you kne</font>w everything.

1117
01:04:45,207 --> 01:04:45,320
<font color="#ff0000">Heh. And you knew</font> everything.

1118
01:04:45,321 --> 01:04:45,434
<font color="#ff0000">Heh. And you knew </font>everything.

1119
01:04:45,435 --> 01:04:45,548
<font color="#ff0000">Heh. And you knew e</font>verything.

1120
01:04:45,549 --> 01:04:45,662
<font color="#ff0000">Heh. And you knew ev</font>erything.

1121
01:04:45,663 --> 01:04:45,777
<font color="#ff0000">Heh. And you knew eve</font>rything.

1122
01:04:45,778 --> 01:04:45,891
<font color="#ff0000">Heh. And you knew ever</font>ything.

1123
01:04:45,892 --> 01:04:46,005
<font color="#ff0000">Heh. And you knew every</font>thing.

1124
01:04:46,006 --> 01:04:46,119
<font color="#ff0000">Heh. And you knew everyt</font>hing.

1125
01:04:46,120 --> 01:04:46,233
<font color="#ff0000">Heh. And you knew everyth</font>ing.

1126
01:04:46,234 --> 01:04:46,347
<font color="#ff0000">Heh. And you knew everythi</font>ng.

1127
01:04:46,348 --> 01:04:46,462
<font color="#ff0000">Heh. And you knew everythin</font>g.

1128
01:04:46,463 --> 01:04:46,576
<font color="#ff0000">Heh. And you knew everything</font>.

1129
01:04:46,577 --> 01:04:46,691
<font color="#ff0000">Heh. And you knew everything.</font>
```

Why reinserting `<font color="#ff0000">Heh. And you knew everything.</font>` twice?